docker-compose.yml

```diff
@@ -230,7 +230,29 @@ services:
         - "dockerhost:${DOCKER_HOST_IP}"
       networks:
         - backend
-### Laravel Horizon ############################################
+
+
+### NODEJS ##############################################
+    nodejs:
+      build:
+        context: ./nodejs
+        args:
+          - LARADOCK_NODE_VERSION=${NODE_VERSION}
+      volumes:
+        - ${APP_CODE_PATH_HOST}:${APP_CODE_PATH_CONTAINER}
+      expose:
```

Comment:
i think this can change to `network_mode: "host"` because range 3000-6000 is too much

Reply:
ok   3000-3010